apps/design/frontend/src/ballots_screen.tsx

```diff
@@ -61,6 +62,20 @@ function BallotDesignForm({
 
   return (
     <Form>
+      <FormField label="Paper Size">
+        <SegmentedControl
```

Comment:
Might experiment with switching this to a radio group

Reply:
Makes sense, looks just a little funky as is.

Reply:
I think radio group looks better

<img width="404" alt="Screen Shot 2023-07-17 at 11 51 17 AM" src="https://github.com/votingworks/vxsuite/assets/530106/09154e0b-e975-4c8e-ab70-0ebfa4eaa617">